vertx-mysql-client/src/main/asciidoc/index.adoc

```diff
@@ -91,6 +92,45 @@ You can easily get one from the pool:
 
 Once you are done with the connection you must close it to release it to the pool, so it can be reused.
 
+=== Unix Domain Socket
+
+Sometimes for simplicity, security or performance reasons, it is required to connect via a Unix Domain Socket.
```

Comment:
can you add a link to how a section of MySQL doc about domain sockets ?

Reply:
Sorry I'm not sure what you mean.

Reply:
Done